docs/links.rst

```diff
@@ -22,19 +22,19 @@
 
 
 .. _Pyramid manual: http://docs.pylonsproject.org/projects/pyramid/en/1.3-branch/index.html
```

Comment:
A few of these are still pointing at `1.3-branch` rather than `latest`, which you didn't comment on. Update?